x/auth/types/client_tx.go

```diff
@@ -1,16 +1,20 @@
 package types
 
 import (
-	"github.com/tendermint/tendermint/crypto"
-
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/legacy"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
 )
 
-// StdTxBuilder wraps StdTx to implement to the context.TxBuilder interface
+// StdTxBuilder wraps StdTx to implement to the context.TxBuilder interface.
+// Note that this type just exists for backwards compatibility with amino StdTx
+// and will not work for protobuf transactions.
```

Comment:
This file has low coverage and we should prob add unit tests on another PR

Reply:
I agree we should cover it better. @anilCSE let's coordinate on a PR to test `TxBuilder` implementations generically.